packages/pg-gateway/src/auth/cert.ts

```diff
@@ -45,21 +45,21 @@ export class CertAuthFlow extends BaseAuthFlow {
 
   async *handleClientMessage(message: BufferSource) {
     if (!this.connectionState.tlsInfo) {
-      yield createBackendErrorMessage({
+      yield BackendError.create({
```

Comment:
I made `createBackendErrorMessage` a class for 2 reasons:
- Easily differentiate it from an unknown error with `instanceof`
- Being able to inspect the data internally before flushing if we ever need that (like the severity)